src/RadixDlt.NetworkGateway.PostgresIntegration/Services/TransactionQuerier.cs

```diff
@@ -129,42 +279,6 @@ public async Task<ICollection<StatusLookupResult>> LookupPendingTransactionsByIn
         return pendingTransactions.Select(pt => new StatusLookupResult(pt.PayloadHash.ToHex(), pt.Status.ToGatewayModel(), pt.LastFailureReason)).ToArray();
```

Comment:
That's beyond the scope of this PR but the original thinking David used in Olympia was to avoid potentially outdated data while reading from read-only replica caused by AWS RDS replication lag.